ui/src/components/AppBar/styles.css

```diff
@@ -0,0 +1,51 @@
+@import "@radix-ui/colors/black-alpha.css";
+@import "@radix-ui/colors/indigo.css";
+@import "@radix-ui/colors/mauve.css";
+@import "@radix-ui/colors/purple.css";
+@import "@radix-ui/colors/violet.css";
+
+.NavigationMenuRoot {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  padding: 1em;
+  width: 100vw;
+  z-index: 1;
+}
+
+.NavigationMenuList {
+  display: flex;
+  justify-content: center;
+  padding: 4px;
+  border-radius: 1px;
+  list-style: none;
+  box-shadow: 0 2px 10px var(--black-a7);
+}
+
+.NavigationMenuTrigger,
+.NavigationMenuLink {
+  padding: 0.6em 1.2em;
+  outline: none;
+  border-radius: 8px;
+  border: 1px solid transparent;
+  font-weight: 500;
+  color: #ffffff;
+  transition: border-color 0.5s;
```

Comment:
can we take this down to 0.25s because 0.5 seems quite long ... either that or change the timing function?